modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling volume snapshot boot sources
+
+Enable volume snapshot boot sources by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
+
+[NOTE]
+====
+It is recommended to use volume snapshots on a storage profile that is proven to scale better when cloning from a single snapshot.
+====
+
+.Prerequisites
+
+* You must have access to a volume snapshot with the operating system image.
```

Comment:
So this can remain prerequisite remains `You must have access to a web page with the operating system image`;
This is because our product takes care of converting the imported source to a VolumeSnapshot.

Maybe a couple of other prerequisites are:
- The storage supports snapshotting
- Storage provider proven to scale better cloning from a single snapshot (not a must, I guess if someone is reading this, they are convinced this is the case)